packages/runtime/src/runtime.ts

```diff
@@ -433,6 +433,17 @@ export class Runtime {
     this.ctx.state.accounts.set(senderAcc.address, senderAcc);
     this.ctx.state.globalApps.set(app.id, senderAcc.address);
 
+    for (const [address, account] of this.store.accounts) {
+      this.ctx.state.accounts.set(address, account);
+      for (const appId of account.createdApps.keys()) {
+        this.ctx.state.globalApps.set(appId, account.address);
+      }
+
+      for (const assetId of account.createdAssets.keys()) {
+        this.store.assetDefs.set(assetId, account.address);
```

Comment:
why are we are operating on `store` before running code ?

Reply:
missed this one. It should be `ctx`